Tests/test_file_tar.py

```diff
@@ -15,21 +15,25 @@ def setUp(self):
             self.skipTest("neither jpeg nor zip support available")
 
     def test_sanity(self):
-        if "zip_decoder" in codecs:
-            tar = TarIO.TarIO(TEST_TAR_FILE, 'hopper.png')
-            im = Image.open(tar)
-            im.load()
-            self.assertEqual(im.mode, "RGB")
-            self.assertEqual(im.size, (128, 128))
-            self.assertEqual(im.format, "PNG")
-
-        if "jpeg_decoder" in codecs:
-            tar = TarIO.TarIO(TEST_TAR_FILE, 'hopper.jpg')
-            im = Image.open(tar)
-            im.load()
-            self.assertEqual(im.mode, "RGB")
-            self.assertEqual(im.size, (128, 128))
-            self.assertEqual(im.format, "JPEG")
+        for codec, file, format in [
```

Comment:
Let's change `file` to something that doesn't shadow the builtin.

Reply:
Okay, I've changed it to `test_path`